data/widgets/zoo.xml

```diff
@@ -0,0 +1,140 @@
+<!-- Aseprite -->
+<!-- Copyright (C) 2018-2025  Igara Studio S.A. -->
+<gui>
```

Comment:
We can disable the i18n warnings as all strings are not translated and we can keep it in that way:
```suggestion
<gui i18nwarnings="false">
```